### PR TITLE
Ft milestone3: No styles

### PR DESCRIPTION
**Pull request by didymus707**
Here, the app component was changed to a class component
Calculate and operate logic were created
Calculate imported operate
App imported Calculate

### CODE REVIEW

**Review by SpaYco — CHANGES_REQUESTED** [review]
# Changes Requested 

Hello Team! 👋 
Awesome job so far!! 👍 you still need to make some changes though! 

## Required Changes ♻️ 

## Since you're not on the v1.1 module, you need to style the project on this milestone.

* [ ] The styled version of the calculator should look like this: [Calculator](https://github.com/microverseinc/project-react-calculator/blob/master/images/calculator.png) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/calculator-style.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896093/))
* [ ] `App`, `Display` and `ButtonPanel` components use flex display mode ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/useFlex.png))
* [ ] calculator width equals 700px ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/700Wide.png))

##### `Display` component

* [ ] has a gray background ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/display-bg.png))
* [ ] has a height og 100px ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/display-height.png))
* [ ] result is presented with white and bold text ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/display-text.png))
* [ ] has a padding from the edges ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/display-padding.png))
* [ ] result is aligned to the right ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/display-alignment.png))

##### `ButtonPanel` component

* [ ] each row's height is 100px ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/rowHeight.png))

##### `Button` component

* [ ] buttons take 25% of container width (except `0` button) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/25Wide.png))
* [ ] buttons have borders ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/ButtonsBoarder.png))
* [ ] buttons text is centered and black ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/TextBlack.png))
* [ ] it accepts `color` and `wide` as an incoming props ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/colorAndWideProps.png))
* [ ] default `color` for button is orange ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/colorProps.png))
* [ ] `wide` is boolean, true means that button take 50% of space (used for `0` button) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/WideProps.png))

### please make the appropriate changes and ask for another code review ✔️ 

Happy Coding!

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Comment by didymus707** [comment]
Yo, you kidding? I'm on v1.1

**Comment by Forison** [comment]
> Yo, you kidding? I'm on v1.1

Hello @didymus707,
 the code review request you submitted does not not suggest that you this project is v1.1. That is why the previous review made the suggestions. When you have doubts it is better to contact the review on their slack page than to leave a message on the `PR`. My suspicion is that you are not selecting the right project when submitting the review request kindly check that. For now I will mark it invalid so you can resubmit under the right project name.

File: src/logic/calculate.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+/* eslint-disable radix */
+/* eslint-disable no-unused-expressions */
```
Comment (bellom) [inline src/logic/calculate.js]: **Inline comment:** 
With the ESLint inline comment, you're are indirectly changing the linter's rules by disabling these rules, let's fix the codebase instead.

If you are using VSCode, you can delete the inline comment and hover over the error(configure eslint checks locally). And you will see your options. Click on the more info button that will take you to a GitHub repo where it is explained why you are getting this error. I find this feature very educational. Please, try to learn and implement the necessary rules so you don't have to keep this disabling line on top of each file. I believe you can do this! 💪😇

File: src/logic/operate.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+/* eslint-disable radix */
+/* eslint-disable no-unused-expressions */
```
Comment (bellom) [inline src/logic/operate.js:2]: **Inline comment:** 
With the ESLint inline comment, you're are indirectly changing the linter's rules by disabling these rules, let's fix the codebase instead.

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @didymus707  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Changes Required! ♻️ 

**Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Review by SpaYco — APPROVED** [review]
# Project Approved

thank you for making the required changes, your project is now approved!

### please don't forget to merge to master, good luck with the next projects!


Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Pull request merged**